crates/heat-sdk-cli/src/cli_commands/run/training.rs

```diff
@@ -0,0 +1,199 @@
+use clap::Parser;
+use colored::Colorize;
+use heat_sdk::{
+    client::{HeatClient, HeatClientConfig, HeatCredentials},
+    schemas::ProjectPath,
+};
+
+use crate::registry::Flag;
+use crate::{
+    commands::{execute_sequentially, BuildCommand, RunCommand, RunParams},
+    context::HeatCliContext,
+    generation::backend::BackendType,
+    logging::BURN_ORANGE,
+    print_info,
+};
+
+#[derive(Parser, Debug)]
+pub struct TrainingRunArgs {
+    /// The training functions to run
+    #[clap(short = 'f', long="functions", value_delimiter = ' ', num_args = 1.., required = true, help = "<required> The training functions to run. Annotate a training function with #[heat(training)] to register it."
```

Comment:
I don't think "<required>" is necessary in the docstrings as there is already a display convention with `[]` for optional args.

Reply:
Edited my message for format error, `<required>` did not appear.